static/src/javascripts/projects/common/modules/identity/upsell/opt-outs/OptOutsList.js

```diff
@@ -6,21 +6,22 @@ import {
     setConsentsInApi,
 } from 'common/modules/identity/upsell/store/consents';
 import type { ConsentWithState } from 'common/modules/identity/upsell/store/types';
+import { ErrorBar, genericErrorStr } from '../error-bar/ErrorBar';
 
 export class OptOutsList extends Component<
     {},
     {
         consents: ConsentWithState[],
         isLoading: boolean,
         hasUnsavedChanges: boolean,
-        hasError: boolean,
+        errors: string[],
```

Comment:
Is this right?

Reply:
Or maybe, should it be like that everywhere else?

Reply:
changing the errors to an array? yep, figured since I was abstracting the errors into a component having the ability to show n errors (even if we don't) was good to have looking into the future